### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by guilgaly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,71 @@
-gatling-maven-plugin-demo-kotlin
-================================
+Gatling plugin for Maven demo project
```
Comment (slandelle) [inline README.md:1]: Please mention Kotlin

File: README.md [hunk 2]
```diff
@@ -1,8 +1,71 @@
...
+=====================================
 
-Simple showcase of a Kotlin maven project using the gatling-maven-plugin.
+A simple showcase of a Maven project using the
+[Gatling plugin for Maven](https://github.com/gatling/gatling-maven-plugin). The full documentation for the plugin can
+be found [on the Gatling website](https://gatling.io/docs/current/extensions/maven_plugin/).
```
Comment (slandelle) [inline README.md:6]: Please don't advertise the GitHub repository. We want users to be rooted to our website, not GitHub.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,71 @@
...
+This project is written in Kotlin, others are available for [Java](https://github.com/gatling/gatling-maven-plugin-demo-java)
+and [Scala](https://github.com/gatling/gatling-maven-plugin-demo-scala).
 
-    $mvn gatling:test
+It includes:
+
+* minimal `pom.xml`
+* latest version of `io.gatling:gatling-maven-plugin` applied
+* sample
+  [Simulation](https://javadoc.io/doc/io.gatling/gatling-core-java/latest/io/gatling/javaapi/core/Simulation.html)
```
Comment (slandelle) [inline README.md:16]: Same: link to the page generated from https://github.com/gatling/gatling/blob/main/src/docs/content/reference/current/core/concepts/index.md#simulation

**Review by slandelle — CHANGES_REQUESTED** [review]
Actually, I wonder if spreading documentation on the website and on GitHub is a good idea. Probably not.

File: README.md (hunk 3 above)
Comment (guilgaly) [reply README.md]: Changed in all the related pull requests.

**Review by slandelle — APPROVED** [review]

**Pull request merged**